spec/forms/password_form_spec.rb

```diff
@@ -4,10 +4,6 @@
   subject { PasswordForm.new(build_stubbed(:user)) }
 
   it do
-    is_expected.to validate_confirmation_of(:password)
-  end
```

Comment:
Can you explain a little more?  Why is `validate_confirmation_of` no longer something we want to check for?